From e4ecf9601e06ea9a12fc8dfc5980ecb1d68523d7 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Wed, 9 Mar 2022 11:41:51 +0200
Subject: [PATCH 011/103] include: configs: Remove CONFIG_SYS_INIT_RAM_*

CONFIG_SYS_INIT_RAM_ADDR and CONFIG_SYS_INIT_RAM_SIZE are not used
on S32 platforms.

Issue: ALB-8576
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 65ccae1260..3f8a4a58c8 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -373,12 +373,9 @@
 
 #define CONFIG_SYS_SDRAM_BASE		PHYS_SDRAM_1
 
-#define CONFIG_SYS_INIT_RAM_ADDR	CONFIG_SYS_DATA_BASE
-#define CONFIG_SYS_INIT_RAM_SIZE	CONFIG_SYS_MEM_SIZE
-
 #define CONFIG_SYS_INIT_SP_OFFSET	(16 * 1024)
 #define CONFIG_SYS_INIT_SP_ADDR \
-	(CONFIG_SYS_INIT_RAM_ADDR + CONFIG_SYS_INIT_SP_OFFSET)
+	(CONFIG_SYS_DATA_BASE + CONFIG_SYS_INIT_SP_OFFSET)
 
 #define CONFIG_SYS_MMC_ENV_DEV		0
 #define MMC_PART_FAT			1
-- 
2.25.1

